models/peep.rb

```diff
@@ -0,0 +1,40 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :timestamp, :message, :user_id, :username
+
+  def initialize(id:, user_id:, username:, timestamp:, message:)
+    @id = id
+    @timestamp = timestamp
+    @message = message
+    @user_id = user_id
+    @username = username
+  end
+
+  def self.all_peeps
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_app_test')
+    else
+      connection = PG.connect(dbname: 'chitter_app')
+    end
```

Comment:
connection to the database can be extracted to a separate class (Step 14 walkthrough of the Bookmarks project goes through this)